notebooks/Read_CoNLL-U_files.ipynb

```diff
@@ -0,0 +1,1116 @@
+{
```

Comment:
<p>Maybe add a <code>%%time</code>  magic to this cell so that the speed of Parquet reading comes across in the static render of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done. </p>